### PR TITLE
Add a new view for each test that failed in all jobs on a payload

### PR DESCRIPTION
**Pull request by dgoodwin**
New Tab on the page to view a particular payload, that lists all tests that failed across all jobs in the payload, how many times, and which jobs it failed in. 

Helpful just to quickly show the tests that may be failing in multiple
jobs.

### CODE REVIEW

File: pkg/sippyserver/server.go [hunk 1]
```diff
@@ -423,6 +423,35 @@ func (s *Server) jsonGetPayloadAnalysis(w http.ResponseWriter, req *http.Request
 	api.RespondWithJSON(http.StatusOK, w, result)
 }
 
+// jsonGetPayloadTestFailures is an api to fetch information about what tests failed across all jobs in a specific
+// payload.
+func (s *Server) jsonGetPayloadTestFailures(w http.ResponseWriter, req *http.Request) {
+	log.Info("hello")
```
Comment (DennisPeriquet) [inline pkg/sippyserver/server.go:429]: nit: not sure you need it -- but harmless.

File: sippy-ng/src/releases/PayloadTestFailures.js [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+
+  if (!isLoaded) {
+    return <p>Loading...</p>
+  }
+
+  if (rows.length === 0) {
+    return <Typography variant="h5">Analysis not found.</Typography>
+  }
+
+  console.log('hello world')
```
Comment (DennisPeriquet) [inline sippy-ng/src/releases/PayloadTestFailures.js:190]: nit: not sure you need it -- but harmless.

**Review by DennisPeriquet — APPROVED** [review]
/lgtm
You can remove the console logs now or later

**Comment by DennisPeriquet** [comment]
and I look forward to use this!

File: sippy-ng/src/releases/PayloadTestFailures.js (hunk 2 above)
Comment (dgoodwin) [reply sippy-ng/src/releases/PayloadTestFailures.js:190]: My bad, thanks!

https://github.com/openshift/sippy/pull/1148

**Pull request merged**